package.json

```diff
@@ -46,12 +46,16 @@
 		"vue3-common.json",
 		"vue-es5.json",
 		"vue-es6.json",
-		"vue2-es5.js",
-		"vue2-es6.js",
-		"vue3-es6.js",
-		"vue-wrappers.js",
+		"vue2-es5.json",
+		"vue2-es6.json",
+		"vue3-es6.json",
 		"yaml.json",
-		"language"
+		"client",
```

Comment:
If supported, I suggest including a trailing slash to clarify that these are directories. You can use `npm pack` to dry-run a release and see the listed files in the command output (or the tarball file itself).